### PR TITLE
Update arm.scad

### PR DESCRIPTION
**Pull request by Nathaniell1**
I rewrote it whole. I also added translate with condition, to prevent overlaping inner circles (translate which make this possible is still there in the comments)

Original version seems too complicated.

### CODE REVIEW

**Comment by syky27** [comment]
Hi, would you mind to keep all of the issues, pull requests and also commit messages in english please?

**Comment by Nathaniell1** [comment]
Hi,
Sure, sorry about that. 

Conditon preventing creating object with 0 holes added.

**Comment by syky27** [comment]
Well your first commit is still in Czech, rebase and push force, than I'll have a look at it.

**Comment by Nathaniell1** [comment]
Well
1) I have no experience with github.
2) I can't find a way how to change my first commit comment.
3) I dont know what you mean here: rebase and push force

**Comment by syky27** [comment]
This should help you how to change history https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History after you succeed you want to `git push --force` to overwrite git trees on server with your local ones.

**Comment by hroncok** [comment]
By private conversation with @Nathaniell1 I see he maid this PR entirely in the browser, so I agreed I would squash the commits while merging (manually).

File: lesson02/arm.scad [hunk 1]
```diff
@@ -1,66 +1,51 @@
...
- * @param off_set vyosetni horni a spodni casti
- * @param thick tloustka ramena
- * @param number_holes pocet der nahore a dole
- * @param hole_radius polomer der v ramenu
- * mezera mezi diramy je jejich radius, to stejne od kraje
- * vyska je brana ze stredu der ke stredu der
- * off set je pocita opet ze stredu der 
- * @author Jakub Průša
- */
-
```
Comment (hroncok) [inline lesson02/arm.scad]: The czech comment section went away?

**Comment by Nathaniell1** [comment]
Czech comment returned.

**Pull request closed without merging**